e2e/tests/session_steps_test.go

```diff
@@ -42,6 +42,10 @@ const (
 	preExistingProofsKey = "preExistingProofsKey"
 )
 
+func (s *suite) TheUserShouldWaitForTheMessageToBeSubmited(module, message string) {
```

Comment:
Please add a `TODO_TECHDEBT: Evaluate if/where/how this function should be reused in other tests`